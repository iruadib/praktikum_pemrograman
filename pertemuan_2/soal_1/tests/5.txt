John Jane
0 0 0