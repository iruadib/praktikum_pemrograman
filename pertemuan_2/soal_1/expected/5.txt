Payslip for Employee
---------------------
Name: John Jane
Tax (20%): Rp0,00
Installment: Rp0,00
Insurance: Rp0,00
Net Salary: Rp0,00
